### ISSUE TITLE
`res.serverError('string')` causes error when using handlebars as view engine

### ISSUE THREAD

**Issue by h0ward** [balderdashy/sails#2334]
To reproduce:
1. [Setup](https://github.com/sails101/using-handlebars#sails-with-handlebars) sails app with Handlebars as view engine
2. Route index to `TestController.js` controller `'/': {controller: 'test'}`
3. Inside `TestController.js` throw  `index: function (req, res) {res.serverError('test')}`
4. `sails lift` and you will see
```
error: Sending 500 ("Server Error") response: 
 test
warn: res.serverError() :: When attempting to render error page view, an error occured (sending JSON instead).  Details:  TypeError: Cannot use 'in' operator to search for 'root' in test
  at initData (/Users/howard/couch-auctmarts/src/printer/node_modules/handlebars/dist/cjs/handlebars/runtime.js:188:28)
  at ret (/Users/howard/couch-auctmarts/src/printer/node_modules/handlebars/dist/cjs/handlebars/runtime.js:130:14)
  at ret (/Users/howard/couch-auctmarts/src/printer/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:422:21)
  at ExpressHandlebars._renderTemplate (/Users/howard/couch-auctmarts/src/printer/node_modules/sails/node_modules/express-handlebars/lib/express-handlebars.js:296:12)
  at ExpressHandlebars.<anonymous> (/Users/howard/couch-auctmarts/src/printer/node_modules/sails/node_modules/express-handlebars/lib/express-handlebars.js:170:21)
  at /Users/howard/couch-auctmarts/src/printer/node_modules/sails/node_modules/express-handlebars/node_modules/promise/core.js:33:15
  at flush (/Users/howard/couch-auctmarts/src/printer/node_modules/sails/node_modules/express-handlebars/node_modules/promise/node_modules/asap/asap.js:27:13)
  at process._tickDomainCallback (node.js:463:13)
 [TypeError: Cannot use 'in' operator to search for 'root' in test]
```

**Comment by CWyrtzen** [comment]
@h0ward is this still an issue?

**Comment by JerryC8080** [comment]
i come cross the same issue

**Comment by zorthgo** [comment]
I am getting the same error message with res.serverError(), res.forbidden(), and res.notFound(). The only one that seems to be working is res.ok().

**Comment by nodeit** [comment]
Any updates? Getting the same error.

**Comment by sailsbot** [comment]
Thanks for posting, @h0ward.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by cappern** [comment]
Same error.. 👍
